sormas-backend/src/main/java/de/symeda/sormas/backend/util/PatchHelper.java

```diff
@@ -0,0 +1,311 @@
+		}
+		postUpdateOrReplace(jsonObject, existingObject);
+
+	}
+
+	private static <T extends HasUuid> void postUpdateOrReplace(@NotNull JsonNode jsonObject, @NotNull T existingObject) {
+		try {
+
+			Iterator<Map.Entry<String, JsonNode>> jsonObjectFieldMap = jsonObject.fields();
+			while (jsonObjectFieldMap.hasNext()) {
+				try {
+					Map.Entry<String, JsonNode> jsonObjectFieldMapEntry = jsonObjectFieldMap.next();
+					JsonNode jsonObjectFieldNode = jsonObjectFieldMapEntry.getValue();
+					Field existingObjectField = getFieldByName(jsonObjectFieldMapEntry.getKey(), existingObject.getClass());
+					if (existingObjectField == null) {
+						throw new RuntimeException("No such field exception! " + jsonObjectFieldMapEntry.getKey());
+					}
+					existingObjectField.setAccessible(true);
+
+					if (jsonObjectFieldNode.isObject()) {
+						updateExistingObject(existingObject, jsonObjectFieldNode, existingObjectField);
+
+					} else if (jsonObjectFieldNode.isArray()) {
+						updateObjectList(existingObject, jsonObjectFieldNode, existingObjectField);
+
+					} else {
+						setFieldValue(existingObject, existingObjectField, jsonObjectFieldNode);
+					}
+
+				} catch (IllegalAccessException e) {
+					e.printStackTrace();
+					Logger logger = LoggerFactory.getLogger(existingObject.getClass());
+					logger.error(String.format("Failed to update object" + existingObject.getClass().getSimpleName() + ". Error: " + e.getMessage()));
+				}
+
+			}
+
+		} catch (Exception e) {
+			e.printStackTrace();
+			throw new RuntimeException(e);
+		}
+	}
+
+	/**
+	 * This method updates the fields of type collections (lists or array).
+	 * 
+	 * @param existingObject
+	 * @param jsonObjectFieldNode
+	 * @param existingObjectField
+	 * @param <T>
+	 * @throws JsonProcessingException
+	 * @throws IllegalAccessException
+	 */
+	private static <T extends HasUuid> void updateObjectList(T existingObject, JsonNode jsonObjectFieldNode, Field existingObjectField)
+		throws JsonProcessingException, IllegalAccessException {
+		Class listElementClass = getParameterizedType(existingObjectField);
+		ArrayList tempNewElementList = new ArrayList();
+
+		for (JsonNode listElement : jsonObjectFieldNode) {
+			addOrReplaceListElement(existingObject, existingObjectField, listElementClass, tempNewElementList, listElement);
+		}
+
+		if (existingObjectField.getType().isAssignableFrom(List.class)) {
+			Object existingObjectFieldInstance = existingObjectField.get(existingObject);
+			List existingElementList = tempNewElementList;
+			if (existingObjectFieldInstance != null) {
+				existingElementList = (List) existingObjectFieldInstance;
+				existingElementList.clear();
+				existingElementList.addAll(tempNewElementList);
+			} else {
+				existingObjectField.set(existingObject, existingElementList);
+			}
+		} else if (existingObjectField.getType().isArray()) {
+			existingObjectField.set(existingObject, tempNewElementList.toArray());
+		}
```

Comment:
what happens if existingObjectField has another type?

It can happen that the guys using the api send wrong data, `postUpdateOrReplace` fails with exception and in this case it's just ignored